code/modules/mob/dead/observer/verbs.dm

```diff
@@ -238,10 +238,12 @@
 	set name = "Toggle Darkness"
 	set category = "Ghost"
 
-	if (see_invisible == SEE_INVISIBLE_OBSERVER_NOLIGHTING)
-		see_invisible = SEE_INVISIBLE_OBSERVER
-	else
-		see_invisible = SEE_INVISIBLE_OBSERVER_NOLIGHTING
+	if (client && client.darkness_planemaster)
+		switch(client.darkness_planemaster.alpha)
+			if(255)		client.darkness_planemaster.alpha = 230
+			if(230)		client.darkness_planemaster.alpha = 180
+			if(180)		client.darkness_planemaster.alpha = 0
+			else		client.darkness_planemaster.alpha = 255
```

Comment:
No single-line ifs/elses is one of our very few style standards. Just do like
```
if(thing)
    stuff
```
instead of 
```
if(thing) stuff
```

Reply:
it looks nicer in a single line but okay, i kneel to the code standard